scripts/zowe-create-MVS-artefacts.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2019, 2019
+################################################################################
+
+# create MVS artefacts 
+script=zowe-create-MVS-artefacts.sh
+echo "<$script>" >> $LOG_FILE 
+echo "Creating MVS artefacts ... "
+
+# Unpax the LOADLIB and SAMPLIB
+echo "Unpax the LOADLIB and SAMPLIB:" >> ${LOG_FILE}
+TEMP_DIR=/tmp/zowe/$script/$$ # temporary coding
```

Comment:
Change to use `$TEMP_DIR` that is the temporary folder for an install to create temporary files that will have guaranteed permission (because the user doing the install creates it) and also is local and will be deleted.